internal/nonkube/client/fs/path_provider.go

```diff
@@ -1,19 +1,17 @@
 package fs
 
-import "fmt"
+import (
+	"github.com/skupperproject/skupper/pkg/nonkube/api"
+)
 
 type PathProvider struct {
 	Namespace string
 }
 
-func (p *PathProvider) getDefaultNamespace() string {
-	return ".local/share/skupper/namespaces/default/input/sources"
-}
-
 func (p *PathProvider) GetNamespace() string {
+	return api.GetHostNamespaceHome(p.Namespace) + "/input/sources"
```

Comment:
suggest replace "/input/sources" with api.InputSiteStatePath and concatenate with path.Join()

Reply:
sorry thought I caught all of these.   Fixed